lib/perl/Genome/Model/Tools/DetectVariants2/Result/DetectionBase.pm

```diff
@@ -405,6 +406,7 @@ sub vcf_result_params {
         aligned_reads_sample => $aligned_reads_sample,
         ($control_aligned_reads_sample? (control_aligned_reads_sample => $control_aligned_reads_sample) : ()),
 
+        #users => $users, #Needed if get() inside get_vcf_result becomes get_with_lock()
```

Comment:
I'm not a fan of the comment but it's definitely good that this won't break silently.